ci.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
Not all distributions install bash to this location.

```suggestion
#!/usr/bin/env bash
```

Reply:
Changed it. Thanks!